hack/KindInstallationREADME.md

```diff
@@ -0,0 +1,66 @@
+<!-- ABOUT THE PROJECT -->
+
+# Running Skydive Operator on Kind cluster
+
+This is a simple deployment guide for the operator on a kind cluster
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+To set up this operator on a kind follow the instructions below:
+
+### Prerequisites
+
+* Make sure you have golang installed on your machine, The go-lang version that this operator was built on is go1.15.7
+  darwin/amd64
+
+* Kind installed
+
+### Modyfing the CRD
+Modify the file config/skydive_v1_skydive.yaml and pick what you wish to deploy (insert true or false in the relevant field), the options are as follows:
+* Skydive agents
+* Skydive analyzer
+    * Service route
+  
+
+### Installation
+
+#### Kind
+Install kind on your machine, use [this](https://kind.sigs.k8s.io/docs/user/quick-start/) quick start guide for help, 
+make sure that your have a running kind cluster and that kubectl commands refer to relevant context.
+
+After installing and running the Kind cluster, make sure the env var KUBECONFIG is set to the correct kubeConfig file,
+it usually defaults to $HOME/.kube/config, run this command:
```

Comment:
I think this part is also relevant for the openshift install, not only Kind. I just run into the issue, with this error in logs : `unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined`. I had no KUBECONFIG set up (I'm connecting to a remote CRC with `oc` ... `oc` works fine but doesn't set up the relevant kubeconfig when I log in, so I had to pull it manually from my remote crc)